app/models/team.rb

```diff
@@ -16,24 +16,24 @@ class Team < ApplicationRecord
 
```

Comment:
Now that you are at it: can we get rid of line 11 as well?

Reply:
Oh yes, I probably can.

I remember I wanted to search for what this could have been once - but then just forgot about it 🙈 

I'll clean this up.